### PR TITLE
add a check for metadata versions of 2 or more to show version selector

### PR DESCRIPTION
**Pull request by ediiotero**
### Description
https://vajira.max.gov/browse/API-2002
Hide version selector if there is only one version of the API

### Process
Added a check for the number of metadata versions

### CODE REVIEW

File: src/containers/documentation/SwaggerDocs.tsx [hunk 1]
```diff
@@ -129,7 +129,9 @@ class SwaggerDocs extends React.Component<ISwaggerDocsProps> {
         url: this.props.docUrl,
       });
       ui.versionActions.setApiVersion(this.props.versionNumber);
-      ui.versionActions.setApiMetadata(this.props.metadata);
+      if (this.props.metadata?.meta.versions.length > 1) {
```
[the remark this answers is not in the document]
Comment (mjlumetta) [reply src/containers/documentation/SwaggerDocs.tsx:132]: not particularly related to the above, but if you feel up to refactoring the state for version metadata in the store and this component, we could cut out a couple of levels. we really only need the versions array, so we could cut out some unnecessary levels to that object.

```
if (this.props.versions?.length > 1) {
  ...
}
```

**Review by mjlumetta — COMMENTED** [review]
looks good! certainly meets the acceptance criteria. i did write up my thoughts from yesterday's working session for discussion. would like to get a little discussion going on that and then complete this PR based on our thoughts there.

File: src/containers/documentation/SwaggerDocs.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ediiotero) [reply src/containers/documentation/SwaggerDocs.tsx]: After reading your post and making the necessary changes, I think the second option is the way to go. Keeping the logic out of the SwaggerDocs file and adding another check to the  ExtendedLayout version select is a better solution. Thanks, this was really helpful 👍

File: src/containers/documentation/SwaggerDocs.tsx [hunk 2]
```diff
@@ -128,7 +128,7 @@ class SwaggerDocs extends React.Component<ISwaggerDocsProps> {
         plugins: [plugins],
         url: this.props.docUrl,
       });
-      ui.versionActions.setApiVersion(this.props.versionNumber);
+      ui.versionActions.setApiVersion(this.props.versionNumber);
```
Comment (mjlumetta) [inline src/containers/documentation/SwaggerDocs.tsx:131]: if you make another push, would you mind reverting this file? looks like it's adding a little unnecessary whitespace

**Review by mjlumetta — APPROVED** [review]
looks great!
